### ISSUE TITLE
[Accessibility][Combobox][Fabric UI]:  Accessibility issues on running keros on multiselect combobox

### ISSUE THREAD

**Issue by mraveesha** [microsoft/fluentui#17452]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: MS Edge and Edition	Windows 10 Enterprise
Version	20H2

### Describe the issue:

Failure on running keros on multiselect Combobox options menu

### Please provide a reproduction of the issue in a codepen:
1. add 'multiSelect' attribute to the combobox component
2. click on the combobox dropdown to open the options menu
3. Run keros

#### Actual behavior:
Keros failed instances: **aria-allowed-attr** and **aria-required-children**

#### Expected behavior:
No keros failed instances

### Documentation describing expected behavior
n/a

**Comment by gouttierre** [comment]
@mraveesha  - Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the [Combobox ](https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox) examples (click “Export to CodePen”) or https://aka.ms/fluentpen.
